cmd/mt-replicator/metrics.go

```diff
@@ -10,6 +11,11 @@ import (
 	"gopkg.in/raintank/schema.v1"
 )
 
+var (
+	metricsBatchSize    = flag.Int("metrics-batch-size", 1000, "number of metrics in each write batch")
+	metricsFetchDefault = flag.Int("metrics-fetch-defaut", 32768, "default number of bytes to fetch with each read")
```

Comment:
it still can be easily misunderstood to mean number of metrics instead of number of bytes though. and also it says `defaut` instead of `default`. My suggestion:
* move the flags to main.go where all of the other flags are, and use newlines to separate global options vs metric relay options vs metricpersist relay options
* call this option `metrics-consumer-fetch-default`